pkg/lifecycle/lifecycle.go

```diff
@@ -496,8 +496,9 @@ type Rule struct {
 
 // Configuration is a collection of Rule objects.
 type Configuration struct {
-	XMLName xml.Name `xml:"LifecycleConfiguration,omitempty" json:"-"`
-	Rules   []Rule   `xml:"Rule"`
+	XMLName                 xml.Name `xml:"LifecycleConfiguration,omitempty" json:"-"`
+	Rules                   []Rule   `xml:"Rule"`
+	ExpiryIgnoreReplication string   `xml:"-,omitempty" json:"-"`
```

Comment:
> `xml:"-,omitempty"`

Dash implies ignore.  Maybe you meant `xml:",omitempty"`?

Reply:
should have used `-`, the new field is a custom setting for minio and will break `get` calls for the lifecycle config with other sdk clients that are aws compliant. 

Further, was thinking about  site replication and  `mc admin cluster bucket export/import that are on older release - sending this as a header allows backward compliance because the header can be ignored